docs/IDENTITY_PROIVIDER.md

```diff
@@ -0,0 +1,191 @@
+            "authorizationUrl": "https://accounts.google.com/o/oauth2/v2/auth",
+            "disableUserInfo": "false",
+            "sendIdTokenOnLogout": "true",
+            "passMaxAge": "false"
+          }
+        }
+    ],
+    ```
+    * note the `${REALM_GOOGLE_IDP_ENABLED}`, `${REALM_GOOGLE_IDP_CLIENT_SECRET}`, and `${REALM_GOOGLE_IDP_CLIENTID}` pieces that are provided by uds-core environment variables
+
+2. Update `uds-core` 
+   1. `uds-core/src/keycloak/chart/templates/secret-kc-realm.yaml`
+
+        ```
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: {{ include "keycloak.fullname" . }}-realm-env
+          namespace: {{ .Release.Namespace }}  
+          labels:
+            {{- include "keycloak.labels" . | nindent 4 }}
+        type: Opaque
+        data:
+          {{- range $key, $value := .Values.realmInitEnv }}
+          {{- if eq (typeOf $value) "bool" }}
+          REALM_{{ $key }}: {{ toString $value | b64enc }}
+          {{- else }}
+          REALM_{{ $key }}: {{ $value | b64enc }}
+          {{- end }}
+          {{- end }}
+        ```
+
+   2. `uds-core/src/keycloak/chart/values.yaml`
+
+        ```
+        # UDS Identity Config Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#override-default-realm
+        realmInitEnv:
+          GOOGLE_IDP_ENABLED: false
+          # Other UDS Identity Config fields that will be used in the realm.json initalization of keycloak
+          # GOOGLE_IDP_CLIENTID: ""
+          # GOOGLE_IDP_CLIENT_SECRET: ""
+        ```
+        * This defines the default values to be used by the IDP, if left unchanged a Google SSO IDP will be created however it will not work as it is disabled and has no client details provided.
+
+   3. `uds-core/src/keycloak/chart/templates/statefulset.yaml`
+
+        ```
+          envFrom:
+            - secretRef:
+                name: {{ include "keycloak.fullname" . }}-realm-env
+        ```
+
+## Override default SSO values
+
+>[!TIP]
+> Additional information about this in the [CUSTOMIZE.md](./CUSTOMIZE.md#override-default-realm)
+
+`uds-identity-config/bundles/uds-bundle.yaml`
+
+```
+    overrides:
+      keycloak:
+        keycloak:
+          variables:
+            - name: KEYCLOAK_CONFIG_IMAGE
+              description: "The keycloak config image to deploy plugin and initial setup configuration"
+              path: configImage
+            - name: GOOGLE_SSO_ENABLED
+              description: "Enable Google SSO IDP"
+              path: realmInitEnv.REALM_GOOGLE_IDP_ENABLED
+              default: true
+            - name: GOOGLE_SSO_CLIENT_ID
+              description: "Set Google SSO Client ID"
+              path: realmInitEnv.REALM_GOOGLE_IDP_CLIENTID
+              default: "{fill in value here}"
+            - name: GOOGLE_SSO_CLIENT_Secret
```

Comment:
```suggestion
            - name: GOOGLE_SSO_CLIENT_SECRET
```

Reply:
just updated this to use the values like you used for testing. Easier to understand and less unnecessary fields/words.